dotCMS/src/main/webapp/html/ng-contentlet-selector.jsp

```diff
@@ -52,9 +52,9 @@
     String baseTypeToAdd = request.getParameter("add");
 
     if (BaseContentType.WIDGET.name().equalsIgnoreCase(baseTypeToAdd)) {
-        contentTypes = APILocator.getContentTypeAPI(user).findByBaseType(BaseContentType.WIDGET, "name", 250,0);
+        contentTypes = APILocator.getContentTypeAPI(user).findByBaseType(BaseContentType.WIDGET, "name", -1,0);
```

Comment:
should it go into a file config property?

Reply:
No I don't think. These are content types, not content. Can't them being THAT many to cause performance issues.